docs/api/Library.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Library
+order: 214
+section: components
+---
+
+```javascript
+var Library = require('sketch/dom').Library
+```
+
+A Sketch Library.
+
```

Comment:
perhaps add a note that the properties are read-only, as I think the expectation we established was that those in this table would be read/write?